packages/temen/src/isArrayBuffer/index.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * value가 ArrayBuffer인지 확인
+ *
+ * @param {*} value 검사할 값
+ * @returns {boolean} ArrayBuffer이면 true 반환 아닌 경우 false 반환
+ * @example
+ *
+ * ```js
+ * isArrayBuffer(new ArrayBuffer(2))
+ * // => true
+ *
+ * isArrayBuffer(new Array(2))
+ * // => false
+ * ```
+ */
+
+function isArrayBuffer(value: any): boolean {
```

Comment:
```suggestion
function isArrayBuffer(value: any) value is ArrayBuffer {
```
로 타입 가딩해주시면 더 좋을 것 같습니다 ㅎㅎ

Reply:
그르네영 써제션 감사합니당~!